platform/ios/NSBundle+MGLAdditions.h

```diff
@@ -2,6 +2,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+void linkBundleCategory();
```

Comment:
These functions should not be public. Will this still work if you declare them in MGLCategoryLoader.m (but still implement them where they are now)?